### PR TITLE
low poly terrain

### PR DESCRIPTION
**Pull request by brnkhy**

### CODE REVIEW

File: sdkproject/Assets/Mapbox/Unity/MeshGeneration/Factories/LowPolyTerrainFactory.cs [hunk 1]
```diff
@@ -0,0 +1,517 @@
...
+		private float _heightModifier = 2f;
+		[SerializeField]
+		private int _sampleCount = 10;
+		[SerializeField]
+		private bool _addCollider = false;
+		[SerializeField]
+		private bool _addToLayer = false;
+		[SerializeField]
+		private int _layerId = 0;
+
```
Comment (david-rhodes) [inline sdkproject/Assets/Mapbox/Unity/MeshGeneration/Factories/LowPolyTerrainFactory.cs]: This factory needs to respect `_useRelativeHeight` (see `TerrainFactory`).

File: sdkproject/Assets/Mapbox/Unity/MeshGeneration/Factories/LowPolyTerrainFactory.cs [hunk 2]
```diff
@@ -0,0 +1,517 @@
...
+		private void GenerateTerrainMesh(UnityTile tile)
+		{
+#if UNITY_5_5_OR_NEWER
+            tile.MeshFilter.mesh.GetVertices(_currentTileMeshData.Vertices);
+            tile.MeshFilter.mesh.GetNormals(_currentTileMeshData.Normals);
+#else
+			_currentTileMeshData.Vertices.Clear();
+			_currentTileMeshData.Vertices.AddRange(tile.MeshFilter.mesh.vertices);
+			_currentTileMeshData.Normals.Clear();
+			_currentTileMeshData.Normals.AddRange(tile.MeshFilter.mesh.normals);
```
Comment (david-rhodes) [inline sdkproject/Assets/Mapbox/Unity/MeshGeneration/Factories/LowPolyTerrainFactory.cs]: You need to recalculate bounds so meshes don't disappear from the camera's view.

File: sdkproject/Assets/Mapbox/Unity/MeshGeneration/Factories/LowPolyTerrainFactory.cs (hunk 1 above)
Comment (brnkhy) [inline sdkproject/Assets/Mapbox/Unity/MeshGeneration/Factories/LowPolyTerrainFactory.cs]: do you think this is necessary? I felt like relative height is only important for realistic views.

**Comment by david-rhodes** [comment]
@isiyu @brnkhy Should we create a new example to demonstrate this? I hate adding new features with no tangible "what does it do."

**Pull request merged**